searching.rst

```diff
@@ -58,24 +71,28 @@ The various placeholders are as follows:
   (e.g. :path:`lib`, :path:`lib32`, :path:`lib64`,
   :path:`lib/x86_64-linux-gnu`...).
 
+:var:`environment-path`:
```

Comment:
Should we differentiate between the build environment and the runtime environment?

Reply:
...what does that mean in this context? `environment-path` in this context is defined (below!) as "a path from ${CPS_PATH}"... which is only inspected when something goes looking for CPS files (e.g. when someone runs `cmake`).

Reply:
Okay, per F2F we'll punt on this, at least for now. Maybe some clarification will happen in the future, but that will be editorial and not functional. Also, I think it's sufficiently obvious that we don't reevaluate what packages to use at runtime. :slightly_smiling_face: